### DISCUSSION TITLE
Compatibility Ejbca with Oracle Dataguard

### DISCUSSION THREAD

**Discussion by jessicahoesl** [Keyfactor/ejbca-ce#165] [Q&A]
Hi,

I would like to know if EJBCA is compatible with Oracle RAC/Dataguard for DB clustering with load balancing. Are there any special requirements needed for this to work? Any problems I could face or is there anything I should pay particular attention to when implementing this?
It would be very helpful, if you could share your view on this or any reference/documentation

Thank you!

**Comment by primetomas** [comment]
Yes it's compatible There are many users using highly available Oracle clusters. There is no specific Oracle documentation maintained by the EJBCA team.
